ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/router_bgp.schema.yml

```diff
@@ -1903,6 +1957,33 @@ keys:
                     type: bool
                   bfd:
                     type: bool
+                    description: Create a bfd session request with default bfd intervals.
+                  bfd_intervals:
+                    type: dict
+                    description: |-
+                      This does not start the bfd session. Set the bfd value to true to create bfd session.
+                      This is used to override the default bfd intervals.
+                    keys:
```

Comment:
You should set `required:true` on all the keys below. Same elsewhere where you have the same data model.